src/api/python/wheels/build_wheel.py

```diff
@@ -0,0 +1,155 @@
+###############################################################################
+# Top contributors (to current version):
+#   Makai Mann
+#
+# This file is part of the cvc5 project.
+#
+# Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+# in the top-level source directory and their institutional affiliations.
+# All rights reserved.  See the file COPYING in the top-level source
+# directory for licensing information.
+# ############################################################################
+#
+# Script for building wheels distribution for pycvc5
+#
+# Example usage (from top-level directory):
+#   python3 ./src/api/python/wheels/build_wheel.py bdist_wheel
+# Configures and builds in directory ./build
+# Creates wheel in ./dist
+#
+##
+
+import os
+import re
+import sys
+import platform
+import subprocess
+import multiprocessing
+import shutil
+import glob
+
+from setuptools import setup, Extension
+from setuptools.command.build_ext import build_ext
+from distutils.version import LooseVersion
+
+def get_project_src_path():
+    # expecting this script to be in src/api/python/wheels
+    # The project source directory is 4 directories up
+    # (5 dirnames because start with file)
+    project_src_path = __file__
+    for i in range(5):
+        project_src_path = os.path.dirname(project_src_path)
+    return os.path.abspath(project_src_path)
+
+
+def get_cvc5_version():
+    project_src_path = get_project_src_path()
+
+    # read CMakeLists.txt to get version number
+    version = dict()
+    str_pattern = 'set\(CVC5_(?P<component>MAJOR|MINOR|RELEASE)\s*(?P<version>\d+)\)'
+    pattern = re.compile(str_pattern)
+    with open(project_src_path + '/CMakeLists.txt', 'r') as f:
```

Comment:
```suggestion
    with open(os.path.join(project_src_path, '/CMakeLists.txt'), 'r') as f:
```

Reply:
Good idea, added in a1f6c9c540e12bdca91526ea28e94ed2eaf545d9.